api.md

```diff
@@ -532,10 +537,10 @@ are before the cut-off point, but only one chunk's worth.
 | `schema_name` | Schema name of the hypertable from which to drop chunks. Defaults to `public`.
 | `cascade` | Boolean on whether to `CASCADE` the drop on chunks, therefore removing dependent objects on chunks to be removed. Defaults to `FALSE`.
 
-The `older_than` parameter can be specified in two ways:
+The `older_than` and `newer_than` parameters can be specified in two ways:
 
 - **interval type**: The cut-off point is computed as `now() -
-    older_than`.  An error will be returned if an INTERVAL is supplied
+    older_than` and similarly `now() - newer_than`.  An error will be returned if an INTERVAL is supplied
```

Comment:
Does it allow something like "Delete all chunks more than 1 day in the future", i.e., to correct for ingested data with incorrect clocks"?  That has come up as a use case...

Reply:
Yes, this works. Updating.

Reply:
In fact, it seems like this old description wasn't really describing this case where you can have older_than greater than now(), I think this needs a rewrite or to be deleted.

Reply:
actually, if you put in an interval type, it will still do now() - newer_than(), but you can do delete all chunks more than 1 day in the future if you do math. I'll add it as an example.